go.work.sum

```diff
@@ -0,0 +1,255 @@
+github.com/Azure/azure-sdk-for-go v46.0.0+incompatible h1:4qlEOCDcDQZTGczYGzbGYCdJfVpZLIs8AEo5+MoXBPw=
```

Comment:
I am trying to Google the answer but my only question is what keeps this `go.work.sum` file updated if say something is changed under `./control-plane` ?

Reply:
I think you call `go work sync` to do this.

Reply:
Could you add this to the contributing doc, please?

Reply:
@t-eckert Is there any particular place in those docs you think would be good. I went looking for where module tidying might be mentioned to add it nearby but that isn't documented?

Reply:
Totally! I think it could go in the [CONTRIBUTING.md](https://github.com/hashicorp/consul-k8s/blob/main/CONTRIBUTING.md#rebasing-contributions-against-main) file in a section below "Rebasing contributions against Main".